DOCUMENTATION.md

```diff
@@ -0,0 +1,271 @@
+
```

Comment:
This is the old commit with the same defects. Update the file with the draft shared in the comments